<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientEditView</class>
 <widget class="QWidget" name="PatientEditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑患者信息</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 整体窗口背景颜色 */
QWidget {
    background-color: #e8f5e9; /* 淡绿色背景 */
    font-family: &quot;Microsoft YaHei UI&quot;, sans-serif; /* 使用微软雅黑字体 */
    font-size: 10pt; /* 默认字体大小 */
}

/* 窗口标题样式 */
QWidget#PatientEditView {
    background-color: #a5d6a7; /* 浅绿色背景 */
    border-radius: 10px; /* 圆角 */
    padding: 10px;
}

/* 标签样式 */
QLabel {
    font-size: 10pt; /* 默认字体大小 */
    font-weight: normal;
    color: #2e7d32; /* 深绿色字体 */
    margin-bottom: 8px;
}

/* 输入框样式 */
QLineEdit, QDateEdit, QComboBox, QSpinBox {
    border: 1px solid #81c784; /* 边框颜色 */
    border-radius: 5px; /* 圆角 */
    padding: 6px;
    background-color: #ffffff; /* 白色背景 */
    font-size: 10pt; /* 默认字体大小 */
    margin-bottom: 10px;
}

/* 聚焦时输入框的样式 */
QLineEdit:focus, QDateEdit:focus, QComboBox:focus, QSpinBox:focus {
    border-color: #388e3c; /* 聚焦时的边框颜色 */
    box-shadow: 0 0 5px rgba(56, 142, 60, 0.5); /* 聚焦时的阴影效果 */
}

/* 按钮样式 */
QPushButton {
    background-color: #66bb6a; /* 绿色背景 */
    color: white;
    border: none;
    border-radius: 5px; /* 圆角 */
    padding: 8px 16px; /* 按钮内边距 */
    font-size: 10pt; /* 默认字体大小 */
    cursor: pointer; /* 鼠标悬停时变成手型 */
    margin-top: 10px;
}

/* 按钮悬停时的样式 */
QPushButton:hover {
    background-color: #388e3c; /* 悬停时的深绿色背景颜色 */
}

/* 布局间距 */
QGridLayout {
    margin: 15px;
    spacing: 8px; /* 控件之间的间距 */
}

/* 控件之间的空隙 */
QSpacerItem {
    background-color: transparent;
}

/* 日期编辑框的样式 */
QDateEdit {
    border: 1px solid #81c784;
    border-radius: 5px;
    padding: 6px;
    font-size: 10pt;
}

/* 下拉框样式 */
QComboBox {
    border: 1px solid #81c784;
    border-radius: 5px;
    padding: 6px;
    font-size: 10pt;
    background-color: #ffffff;
}

/* 下拉框下拉列表样式 */
QComboBox::drop-down {
    border: none;
}

/* 下拉框选项样式 */
QComboBox QAbstractItemView {
    background-color: #ffffff;
    border: 1px solid #81c784;
    padding: 6px;
}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>83</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>体重：</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="dbWeight">
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="dbEditName"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="dbEditID"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>身高：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="dbEditMobile"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>手机号：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>性别：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDateEdit" name="dbDateEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="dbEditIDCard"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>出生日期：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>身份证：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="dbSpinHeight">
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="dbComboSex">
       <item>
        <property name="text">
         <string>男</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>女</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>创建时间</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="dbCreatedTimestamp"/>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>178</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>82</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
